src/panels/NightSkyPanel/NightSkyPanel.tsx

```diff
@@ -0,0 +1,50 @@
+import { Container, Divider, Group, Space, Tabs } from '@mantine/core';
+import { useElementSize } from '@mantine/hooks';
+
+import { NightSkyLocationTab } from './tabs/NightSkyLocationTab';
+import { NightSkyMarkingsTab } from './tabs/NightSkyMarkingsTab';
+import { NightSkySolarSystemTab } from './tabs/NightSkySolarSystemTab';
+import { NightSkyStarsTab } from './tabs/NightSkyStarsTab';
+import { NightSkySunTab } from './tabs/NightSkySunTab';
+import { NightSkyTimeTab } from './tabs/NightSkyTimeTab';
+
+export function NightSkyPanel() {
+  const { ref } = useElementSize();
+
+  return (
+    <Container fluid>
+      <Tabs variant={'outline'} radius={'md'} defaultValue={'markings'}>
+        <Tabs.List ref={ref}>
+          <Tabs.Tab value={'markings'}>Markings</Tabs.Tab>
+          <Tabs.Tab value={'time'}>Time</Tabs.Tab>
+          <Tabs.Tab value={'location'}>Location / View</Tabs.Tab>
+          <Tabs.Tab value={'stars'}>Stars</Tabs.Tab>
+          <Tabs.Tab value={'solarsystem'}>Solar System</Tabs.Tab>
+          <Tabs.Tab value={'sun'}>Sun</Tabs.Tab>
+        </Tabs.List>
+        <Divider size={'xl'}></Divider>
+        <Space h={'lg'}></Space>
+        <Group bg={'dark'} p={'lg'}>
```

Comment:
Please remove the background, as it is inconsistent with other tabs and messes with the contrasts. I also suggest to remove some spacing
```suggestion
        <Group p={'md'}>
```